custom/covid/tasks.py

```diff
@@ -77,20 +78,21 @@ def get_users_for_priming(domain, sync_window, sync_cutoff, min_case_load):
     """
     assert sync_window > sync_cutoff
 
-    users_synced_in_window = set(
+    base_query = (
         SyncLogSQL.objects.values("request_user_id", "user_id")
         .filter(
             domain=domain,
-            date__gt=sync_window,
             is_formplayer=True,
+        ).exclude(auth_type=FORMPLAYER)  # ignore syncs that were done by SMS or by this task
+    )
+
+    users_synced_in_window = set(
+        base_query.filter(
+            date__gt=sync_window,
             case_count__gt=min_case_load
         ).distinct()
     )
 
-    users_synced_since_cutoff = set(
-        SyncLogSQL.objects.values_list("request_user_id", "user_id")
-        .filter(domain=domain, date__gt=sync_cutoff, is_formplayer=True)
-        .distinct()
-    )
+    users_synced_since_cutoff = set(base_query.filter(date__gt=sync_cutoff).distinct())
```

Comment:
Are there tests for this code? Should this change be tested?

Reply:
there are no tests at present. I can write some in a separate PR (this code is only called in the mgmt command at the moment since there are no domains in the FF)